File: app/src/main/java/com/asmat/rolando/popularmovies/database/AppDatabase.java
package com.asmat.rolando.popularmovies.database;

import android.arch.persistence.room.*;

@Database(entities = {FavoriteMovie.class, Movie.class, WatchLaterMovie.class}, version = 2)
public abstract class AppDatabase extends RoomDatabase {
    public abstract MoviesDAO moviesDAO();
}

File: app/src/main/java/com/asmat/rolando/popularmovies/database/AppDatabase.kt
package com.asmat.rolando.popularmovies.database

import androidx.room.*

import com.asmat.rolando.popularmovies.database.entities.FavoriteMovie
import com.asmat.rolando.popularmovies.database.entities.WatchLaterMovie

@Database(entities = [FavoriteMovie::class, WatchLaterMovie::class], version = 3)
abstract class AppDatabase : RoomDatabase() {
    abstract fun moviesDAO(): MoviesDAO
}